cid-redirects.json

```
@@ -2375,6 +2375,7 @@
   "/cid/5421": "/docs/search/search-query-language/search-operators/fillmissing",
   "/cid/5422": "/docs/search/time-compare",
   "/cid/12356": "/docs/integrations/sumo-apps/log-analysis-quickstart",
+  "/cid/12357": "/docs/integrations/sumo-apps/sumo-logic-sample-data-astronomy",
```

For Sumo Logic apps URLs, we omit the "sumo-logic-" prefix to keep it clean. See [examples](https://help.sumologic.com/docs/integrations/sumo-apps).

```suggestion
  "/cid/12357": "/docs/integrations/sumo-apps/sample-data-astronomy",
```